src/services/jwt.service.js

```diff
@@ -0,0 +1,37 @@
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY);
```

Comment:
Ensure that the `user` object passed to `jwt.sign` does not contain sensitive information such as passwords. Consider extracting only necessary fields (e.g., user ID, email) before signing the token.